ci/jjb/jobs/pulp-restore.yaml

```diff
@@ -0,0 +1,43 @@
+- job-template:
+    name: 'pulp-{pulp_version}-{pulp_build}-restore-{os}'
+    node: '{os}-np'
+    properties:
+        - qe-ownership
+    scm:
+        - pulp-packaging-github
+    wrappers:
+        - config-file-provider:
+            files:
+                - file-id: rhn_credentials
+                  variable: RHN_CREDENTIALS
+        - inject:
+            properties-content: |
+                OS={os}
+                PULP_BUILD={pulp_build}
+                PULP_VERSION={pulp_version}
+    triggers:
+        - reverse:
+            jobs: 'pulp-2.11-stable-backup-rhel6'
```

Comment:
@kdelee you don't need to put on the name. For example:

https://github.com/pulp/pulp_packaging/blob/master/ci/jjb/jobs/projects.yaml#L61 and https://github.com/pulp/pulp_packaging/blob/master/ci/jjb/jobs/pulp-dev.yaml#L20